ExchangeWebServices/Source/MS-OXWSCONT/TestSuite/TestSuiteBase.cs

```diff
@@ -988,6 +1011,22 @@ protected void VerifyChildElementInContactItemTypeComplexType(ContactItemType re
                 79,
                 @"[In t:ContactItemType Complex Type] WeddingAnniversary element: Contains the wedding anniversary date of a contact (2).");
 
+            // Add the debug information
+            Site.Log.Add(LogEntryKind.Debug, "Verify MS-OXWSCONT_R7901");
+
+            // Verify MS-OXWSCONT requirement: MS-OXWSCONT_R78

```

Comment:
MS-OXWSCONT_R78 should be MS-OXWSCONT_R7901

Reply:
Have been updated.